frontend/public/locales/en/translation.json

```diff
@@ -1103,9 +1103,9 @@
     "credentialsForm.vCenter.wizardTitle": "Enter the VMware credentials",
     "credentialsForm.vsphereDiskType.labelHelp": "The disk provisioning method. This value defaults to the vSphere default storage policy if not set.",
     "credentialsForm.vsphereDiskType.placeholder": "Select the vSphere disk type",
-    "credentialsForm.vsphereFolder.labelHelp": "The absolute path of the vSphere folder to use, for example: /DATACENTER/vm/FOLDER",
+    "credentialsForm.vsphereFolder.labelHelp": "The absolute path (case-sensitive) of the vSphere folder to use, for example: /DATACENTER/vm/FOLDER",
```

Comment:
Change this to 2 sentences, so:

"The absolute path (case-sensitive) of the vSphere folder to use. For example: /DATACENTER/vm/FOLDER"